packages/adaptive-ui/src/core/modules/types.ts

```diff
@@ -426,3 +426,44 @@ export const stylePropertyPaddingAll = [
  * @public
  */
 export type StyleRules = Array<StyleRule>;
+
+/**
```

Comment:
I think we don't want to add these here yet since we kept them local to the other package for now. We'll merge something similar in after this goes through.

Reply:
We can consolidate later but they're needed here now because [this function](https://github.com/Adaptive-Web-Community/Adaptive-Web-Components/pull/200/files/34ee33b3a20d18f776f85a2f1a56fcac345b998d#diff-238e557a50522594168dcf548e874d5b03f8c0906505265540e518c2496c8411R204) uses them.

Reply:
I see. I didn't notice that moved into the main aui package.